src/webhook_launcher/webhook.conf

```diff
@@ -102,13 +102,32 @@ ldap_base_dn = ou=people,dc=example,dc=com
 ; filter associated with above ldap_base_dn. Do not remove the default, modify it.
 ldap_filter = (uid=%(user)s)
 
+; Do we need to bind to search for a user?
+; AUTH_LDAP_BIND_DN/PW options
+; ldap_bind_dn =
+; ldap_bind_pw =
+
 ; attribute where user emails are stored
 ldap_mail_attr = mail
 ; attribute where users' first names are stored
 ldap_fname_attr = givenName
 ; attribute where users' last names are stored
 ldap_lname_attr = sn
 
+; Support for LDAP groups
+; The group type for AUTH_LDAP_GROUP_TYPE defaults
+; to NestedActiveDirectoryGroupType:
+; ldap_group_type = NestedActiveDirectoryGroupType
+;
+; You must specify where groups are found. Unless this is given
+; no further group settings are done.
+; ldap_group_search_dn = OU=My_Groups,DC=example,DC=com
```

Comment:
because I copied the working example and just replaced the real values
it's case insensitve